### DISCUSSION TITLE
Trying to upgrade to geographiclib 1.52

### DISCUSSION THREAD

**Discussion by cffk** [Homebrew/discussions#1726] [Everyday usage]
I just made a new release of geographiclib 1.52.  I went through my usual steps of updating `Formula/geographiclib.rb` in https://github.com/Homebrew/homebrew-core  However when I do `brew upgrade geographiclib`, I get

```
$ brew upgrade geographiclib
==> Upgrading 1 outdated package:
geographiclib 1.51 -> 1.52
==> Upgrading geographiclib
  1.51 -> 1.52 

==> Downloading https://ghcr.io/v2/homebrew/core/geographiclib/manifests/1.52-1
##O=#  #                                                                      
curl: (22) The requested URL returned error: 404 
Error: Failed to download resource "geographiclib_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/geographiclib/manifests/1.52-1
```

I've no idea what the `ghcr.io` download is about.  Any help is appreciated.

**Comment by carlocab** [comment]
It's trying to download a bottle, but one doesn't exist yet. Are you trying to build it from source? If so, you want the flag `--build-from-source`.

**Reply by cffk** [reply]
Thanks. I have things working now!

**Comment by nandahkrishna** [comment]
The formula hasn't been upgraded to `1.52` in the Homebrew/homebrew-core repository. When you run `brew upgrade <formula>`, it tries to upgrade the formula using the Bottle (binary package) that we build for it in our CI. Since we haven't built `1.52` (as you only probably updated the formula in your local clone of homebrew-core), you're seeing this error.

**Reply by cffk** [reply]
Thanks. I have things working now!
